### PR TITLE
[AR] Translation of 14-2.md

### PR DESCRIPTION
**Pull request by anasselhoud**

### CODE REVIEW

**Review by AliEl-01 — COMMENTED** [review]
Hi Anas
I think the translation will be better If the title is translated 😅 
TnQ

**Comment by anasselhoud** [comment]
> Hi Anas
> I think the translation will be better If the title is translated 😅
> TnQ

@AliEl-01 Yeah, forgot that. Thanks for the notice.

File: docs/ar/week14/14-2.md [hunk 1]
```diff
@@ -0,0 +1,491 @@
+---
+lang-ref: ch.14-2
+lang: ar
+lecturer: Yann LeCun
+title: الأساليب المبنية على الطاقة الرسومية
+authors: Yada Pruksachatkun, Ananya Harsh Jha, Joseph Morag, Dan Jefferys-White, and Brian Kelly
+date: 4 May 2020
+date: 15 Sep 2020
```
Comment (Atcold) [inline docs/ar/week14/14-2.md:8]:
```suggestion
translation-date: 15 Sep 2020
```

**Review by Atcold — CHANGES_REQUESTED** [review]
You need to update `docs/_config.yml` as well.

**Comment by anasselhoud** [comment]
@Atcold Done. Ready to be merged, I suppose.

**Review by AliEl-01 — COMMENTED** [review]
THE TITLE could be translated in a better way! I see that
 " أساليب الرسم البياني القائم على الطاقة " 
OR
 " أساليب الرسومية القائمة على الطاقة " 
It will be better as translation!

**Review by Atcold — APPROVED** [review]
Great, merging now.
We can update title and other stuff during the review process.

**Comment by anasselhoud** [comment]
> THE TITLE could be translated in a better way! I see that
> " أساليب الرسم البياني القائم على الطاقة "
> OR
> " أساليب الرسومية القائمة على الطاقة "
> It will be better as translation!

@AliEl-01  The second proposition is 'google translate' generated, not accurate at all. There is nothing called أساليب الرسومية in Arabic. I can go with the first one even if I am not convinced.
As I always say, this is a technical article. You should translate it into a technical article. So knowing the context is important to make the translation understanble technically not just languistically. 
I believe we should leave the reviewing part to someone who has more knowledge in both Deep Learning and Arabic (I study DL and it's my last year studying and even though I don't feel competent to review a whole work), you can eventually discuss it all together in Slack group.

**Pull request closed without merging**